lib/dbservice/profile/teacher_profile.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Dbservice.Profiles.TeacherProfile do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Dbservice.Profiles.UserProfile
+  alias Dbservice.Users.Teacher
+
+  schema "teacher_profile" do
+    field(:teacher_id, :string)
+    field(:school, :string)
+    # should i go with program_manager_id instead
+    field(:program_manager, :string)
+    field(:avg_rating, :decimal)
+    # add plio, attendance, tests, teacher obs data later
+
+    timestamps()
+
+    belongs_to(:user_profile, UserProfile)
+    belongs_to(:teacher, Teacher, foreign_key: :teacher_fk)
```

Comment:
Same question as above

Reply:
there's no teacher_fk_id.. there's just teacher_fk. earlier it was teacher_id, i renamed it for consistency. deleted and recreated index too

Reply:
uuid in previous table --> teacher_id in current table (string)
teacher_id in previous table --> teacher_fk in current table (int, fk to teacher table)